pkg/database/repository.go

```diff
@@ -65,17 +65,17 @@ func (r repository) FindBySlug(slug string) (*model.Database, error) {
 	return d, err
 }
 
-func (r repository) Lock(id, instanceId, userId uint) (*model.Lock, error) {
+func (r repository) Lock(databaseId, instanceId, userId uint) (*model.Lock, error) {
```

Comment:
Should we stick to `databaseID`? our field names do seem to follow https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I guess we should but it's only a handful of places that we actually do